NRediSearch/Client.cs

```diff
@@ -138,6 +138,39 @@ public SearchResult Search(Query q)
             return new SearchResult(resp, !q.NoContent, q.WithScores, q.WithPayloads);
         }
 
+        /// <summary>
+        /// Return Distinct Values in a TAG field
+        /// </summary>
+        /// <param name="fieldName">TAG field name</param>
+        /// <returns>List of TAG field values</returns>
+        public RedisValue[] TagVals(string fieldName)
+        {
+            var args = new List<object>();
+            args.Add(_boxedIndexName);
+            args.Add(fieldName);
+
+            var resp = (RedisValue[])DbSync.Execute("FT.TAGVALS", args);
```

Comment:
if the number of args is known and fixed: you don't need to use a list - there should be a `params object[]` overload, which means this should work?

    var resp = (RedisValue[])DbSync.Execute("FT.TAGVALS", _boxedIndexName, fieldName);